general_standard/relations_to_other_standards.adoc

```diff
@@ -0,0 +1,18 @@
+= Relations to other standards
+
+== Positioning of Open Simulation Interface within ASAM activities
+
+OSI is part of the ASAM simulation standards that focus on simulation data for the automotive environment.
+Next to OSI, ASAM Provides other standards for the simulation domain, like OpenDRIVE, OpenSCENARIO and OpenCRG.
+
+OpenDRIVE defines a storage format for the static description of road networks.
+In combination with OpenCRG it is possible to add very detailed road surface descriptions to the road network.
+OpenDRIVE and OpenCRG only contains static content.
+To add dynamic content OpenSCENARIO is needed.
+Combined all three standards provide a scenario-driven description of traffic simulation that contains static and dynamic content.
```

Comment:
Maybe this sentence can be reworked, it is a bit hard to read and understand

Reply:
This is a standard text, so I can only make minor changes.
However, I will relay your feedback.
Maybe we implement the changes in the context of #35.